rides/models.py

```diff
@@ -98,15 +100,17 @@ class Rider(db.Model):
     username = db.Column(db.String(80), nullable=False)
     name = db.Column(db.String(50), nullable=False)
     car_id = db.Column(db.Integer, db.ForeignKey('cars.id'), nullable=False)
-    slack = db.Column(db.String, nullable=False) # ADD TO MYSQL
-    email = db.Column(db.String, nullable=False) # ADD TO MYSQL
+    slack = db.Column(db.String, nullable=False)
+    email = db.Column(db.String, nullable=False)
+    acc_type = db.Column(db.String, nullable=False)
```

Comment:
I think I understand what you're saying. Instead of the same 6 sections in riders, we could just input the user object into the rider. The only things I see against it is Rider would then receive a first_name and last_name variable instead of a full_name (but that can easily be solved later) and it receives the users picture that it'll Never use